src/error/default-error-handler.ts

```diff
@@ -1,9 +1,12 @@
-import { Request } from '../request'
+import { Request, RequestEvent } from '../request'
 import { Response } from '../response'
 import { CustomErrorHandler } from './custom-error-handler'
 
-export const DefaultErrorHandler: CustomErrorHandler = (e: Error, request: Request): Response => {
-  console.error(e)
+export const DefaultErrorHandler: CustomErrorHandler = (
+  error: Error,
+  request: Request | RequestEvent
+): Response => {
+  console.error({ request, error })
```

Comment:
Shouldn't logging this stuff be for the user to do

Reply:
yes this is just the default error handler in case one is not supplied. Users should supply their one error handler.